Bio/Blast/NCBIStandalone.py

```diff
@@ -985,7 +985,8 @@ def title(self, line):
     def length(self, line):
         # e.g. "Length = 81" or more recently, "Length=428"
         parts = line.replace(" ", "").split("=")
-        assert len(parts) == 2, "Unrecognised format length line"
+        if len(parts) != 2:
+            raise ValueError("Unrecognised format length line: %s" % line)
```

Comment:
Can you use ``%r`` for the Python object representation here? That tends to be more useful for seeing things like tabs and carriage returns explicitly.